### ISSUE TITLE
Failed to install extension from docker marketplace (2jun2023)

### ISSUE THREAD

**Issue by RarogCmex** [nebula-contrib/nebulagraph-docker-ext#25]
I've just tried to install extension from docker desktop marketplace and I have got this:

`Failed to install extension: installation could not be completed due to: executing '/Applications/Docker.app/Contents/Resources/bin/docker --context desktop-linux compose -p weygu_nebulagraph-dd-ext-desktop-extension -f extensions/weygu_nebulagraph-dd-ext/vm/docker-compose.yaml up -d' : exit status 1: studio Pulling acb7b36cc57a Already exists d2c287eba8ed Already exists 7d3e39d16c58 Already exists studio Pulled Error response from daemon: content digest sha256:86c2a4f793e932b1b82fc2819bff76cff713a486f28d6949655fdf12e65479f1: not found`

Docker Desktop version 4.20 on apple mac m1 (arm64).
It might be temporary docker desktop internal issues. To be honest, we have the software we deserve :(

**Comment by RarogCmex** [comment]
I'm keep continiously failing to install. So it's issue with extension.

**Comment by wey-gu** [comment]
> I'm keeping continiously fail to install. So it's issue with extension.

Thanks, @RarogCmex for the issue, it's strange that I(on m1 macOS, too) cannot reproduce the issue, will look into it later.

Also would you mind sharing this printout?

```
# check images
docker images 
docker --debug extension install weygu/nebulagraph-dd-ext:0.4.8
```
I just pruned images locally and retried fresh installation and didn't encounter issues T__T.

**Comment by wey-gu** [comment]
BTW. Are you using the docker hub or a mirror, please?

**Comment by RarogCmex** [comment]
That's kinda weird. I used direct connection for docker hub. I'm from
Russia, so docker.io should be unblocked and accessible.

пн, 5 июн. 2023 г., 06:46 Wey Gu ***@***.***>:

**Comment by RarogCmex** [comment]
@wey-gu Yeah, I found the issue. I has "use containerd for pulling and storing images" in the docker settings enabled. When I disable this setting, extension has been installed succesfully. However, I'm wondering why it affects.

**Comment by RarogCmex** [comment]
![Screen](https://github.com/nebula-contrib/nebulagraph-docker-ext/assets/20739233/7481048d-1509-4c13-9590-1bdf2d652120)

**Comment by wey-gu** [comment]
> @wey-gu Yeah, I found the issue. I has "use containerd for pulling and storing images" in the docker settings enabled. When I disable this setting, extension has been installed succesfully. However, I'm wondering why it affects.

Thanks a lot for finding the reproduce! I'll look into this later 🫡
